packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -75,11 +81,22 @@ export default class QuanticSort extends LightningElement {
   schemas = {};
 
   labels = {
-    sortBy,
     relevancy,
     newest,
     oldest,
     invalidCustomSortConfig,
+    sortBy,
+  };
+
+  variants = {
+    default: {
+      name: 'default',
+      label: sortBy,
+    },
+    wide: {
+      name: 'wide',
+      label: undefined,
+    },
   };
 
   connectedCallback() {
```

Comment:
It could be worth adding validation here:

```
if(!Object.values(SORT_VARIANTS).includes(this.variant)) {
  console.warn(`Unsupported variant specified in QuanticSort component, defaulting to ${SORT_VARIANTS.DEFAULT}.`);
}
```